pkg/restic/common.go

```diff
@@ -17,11 +17,15 @@ limitations under the License.
 package restic
 
 import (
+	"context"
 	"fmt"
 	"os"
 	"strings"
 	"time"
 
+	"sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
group imports with import block from L-29-32

Reply:
I didn't think we were manually fussing with where imports go?